app/Main.hs

```diff
@@ -7,7 +7,5 @@
 
 module Main (main) where
 
-import Lib
-
 main :: IO ()
-main = someFunc
```

Comment:
Why 😆

Reply:
someFunc does not exist anymore, so i remove the call of it